docs/topics/class-based-views/mixins.txt

```diff
@@ -538,7 +538,7 @@ write our own ``get_context_data()`` to make the
             context["form"] = AuthorInterestForm()
             return context
 
-Then the ``AuthorInterestForm`` is a :class:`FormView`, but we have to bring in
+Then the ``AuthorInterestFormView`` is a :class:`FormView`, but we have to bring in
```

Comment:
Wrap docs at 79 chars:
```diff
diff --git a/docs/topics/class-based-views/mixins.txt b/docs/topics/class-based-views/mixins.txt
index 2603c7d447..c51c03bd80 100644
--- a/docs/topics/class-based-views/mixins.txt
+++ b/docs/topics/class-based-views/mixins.txt
@@ -538,11 +538,11 @@ write our own ``get_context_data()`` to make the
             context["form"] = AuthorInterestForm()
             return context
 
-Then the ``AuthorInterestForm`` is a :class:`FormView`, but we have to bring in
-:class:`~django.views.generic.detail.SingleObjectMixin` so we can find the
-author we're talking about, and we have to remember to set ``template_name`` to
-ensure that form errors will render the same template as ``AuthorDetailView``
-is using on ``GET``::
+Then the ``AuthorInterestFormView`` is a :class:`FormView`, but we have to
+bring in :class:`~django.views.generic.detail.SingleObjectMixin` so we can find
+the author we're talking about, and we have to remember to set
+``template_name`` to ensure that form errors will render the same template as
+``AuthorDetailView`` is using on ``GET``::
 
     from django.http import HttpResponseForbidden

```

Reply:
A bit unfamiliar with Github, sorry for this basic question: Did you already make the necessary change(wrapping line at 79 chars) yourself or that was just a comment and I should do it with a new commit to this PR?

Reply:
> Did you already make the necessary change

No, because you didn't allow for edits from maintainers (there is a checkbox on the right-side).

Reply:
I did. "Allow edits by maintainers" is checked.

Reply:
Pushed.